documentation/pom.xml

```diff
@@ -185,6 +185,151 @@
+                                        <include name="server/runtime/target/classes/schema/*-${infinispan.core.schema.version}.xsd"/>
+                                        <include name="counter/target/classes/schema/*-${infinispan.core.schema.version}.xsd"/>
+                                        <include name="lock/target/classes/schema/*-${infinispan.core.schema.version}.xsd"/>
+                                    </fileset>
+                                </path>
+                                <pathconvert pathsep=" " property="xsd.fileset" refid="xsd.fileset.path"/>
+                                <echo level="info" message="XSDs: ${xsd.fileset}"/>
+                                <java classname="org.infinispan.tools.xsd.XSDoc">
+                                    <arg value="-o"/>
+                                    <arg value="${project.build.directory}/site/configdocs"/>
+                                    <arg value="-s"/>
+                                    <!-- file contains only abstract types. Skip rendering to HTML-->
+                                    <arg value="infinispan-cachestore-jdbc-common-config-${infinispan.core.schema.version}.xsd"/>
+                                    <arg line="${xsd.fileset}"/>
+                                    <classpath>
+                                        <pathelement path="${plugin_classpath}"/>
+                                    </classpath>
+                                </java>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>xsdocupload</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <skip>${skipArtifactUpload}</skip>
+                            <target>
+                                <echo level="info" message="Uploading Schema docs to ${upload.docsDir}"/>
+                                <exec dir="${project.build.directory}/site" executable="rsync" failonerror="${rsync.failonerror}">
+                                    <arg value="-rvm"/>
+                                    <arg line="--protocol=29"/>
+                                    <arg line="-e &quot;ssh -i ${upload.keyFile} -p ${upload.remotePort}&quot;"/>
+                                    <arg value="configdocs"/>
+                                    <arg value="${upload.docsDir}"/>
+                                </exec>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>create-logging-report</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>install</phase>
```

Comment:
I had to change this one to `package` (I guess any phase before `install` would work) otherwise it runs too late and I got 
```
[ERROR] Failed to execute goal org.codehaus.mojo:xml-maven-plugin:1.0.2:transform (generate-logging-html) on project infinispan-docs: Failed to transform input file /home/pedro/Work/infinispan/main/documentation/logs.html: The file or directory file:/home/pedro/Work/infinispan/main/documentation/target/report/tmp does not exist -> [Help 1]
```